docs/reference/components/sinks/amqp.cue

```diff
@@ -0,0 +1,106 @@
+package metadata
+
+components: sinks: amqp: {
+	title: "Amqp"
+
+	classes: {
+		commonly_used: true
+		delivery:      "at_least_once"
+		development:   "stable"
```

Comment:
Could we label this `beta` for now? We usually like to let users test it out more before we promote it to `stable`.